docs/preview/yamlgettingstarted-triggers.md

```diff
@@ -32,8 +34,41 @@ trigger:
 
 Note, path filters are only supported for Git repositories in VSTS.
 
-## CI is opt-out
+### CI is opt-out
 
 Continuous integration builds can be turned off by specifying `trigger: none`
 
 Optionally, the triggers can be managed from the web definition editor, on the Triggers tab.
+
+## Pull request triggers (currently for GitHub repositories only)
+
+Pull request builds are on by default for all branches. Optionally an explicit list of branch filters can
+be specified in the YAML file.
+
+When a PR is updated, the YAML file from the target branch is used to evaluate the branch filters.
+
+And the workflow defined in the YAML file from the merge branch, is used to execute the build.
+
+### Simple PR trigger syntax
+
+A simple list of branches can be specified in the file, to control which branches trigger a PR build.
+
+For example, a simple list of inclusive branch filters may look like:
+
+```yaml
+trigger:
+- master
```

Comment:
I have a tough time with the filters.  What does this mean if the filters don't line up between two different branches?  When branch info is encoded into files in a repo, we're duplicating data that the repo is already telling us. The existence of a pr trigger in a branch tells us that the job should run for that branch

Now, as far as what branches VSTS should watch for yaml config as code jobs, that makes a lot of sense to make branch specific, but should probably live outside the source control.